### DISCUSSION TITLE
Unable to export a model with bilinear terms using "export."

### DISCUSSION THREAD

**Discussion by keeping118** [yalmip/YALMIP#1446] [General discussion] [answered]
Dear professor.
When exporting the model, it seems that models with bilinear terms cannot be exported.Here is a simplified example:
```
sdpvar x y
Model = export([x >= 0, x*y == 1], [],sdpsettings('solver','gurobi'))
```
```
Model =

     []
```
I would like to know if this is an issue with my parameter settings or if YALMIP does not support the export of such models. Your guidance would be greatly appreciated.

**Comment by johanlofberg** [comment, marked as answer]
Install the develop branch as it works there

```
>> M = export([x*y == 1],x+y,sdpsettings('solver','gurobi'));M.quadcon

ans = 

  struct with fields:

       Qc: [2×2 double]
        q: [2×1 double]
      rhs: 1
    sense: '='
```
